docs/tutorial/introduction.md

```diff
@@ -10,13 +10,13 @@ support different levels of use-cases. Each successive layer exposes additional
 abstractions and offers more fine-grained control.
 
 - As a starting point, the new [`inline` feature](inline.md) allows some abstractions (values and methods) to be marked as statically reducible. 
-  It can be used to optimize performance by specializing code and also provides the entry point for macros.
+  It provides the entry point for macros and other metaprogramming utilities.
```

Comment:
@nicolasstucki I thought I mention the first use case first, since this is what the inline tutorial is mostly about.

Reply:
We do not want users to use `inline` for performance reasons as most users try to use it as if this was C++ and ignoring how the JVM works.